src/auth/googleauth.ts

```diff
@@ -659,18 +658,18 @@ export class GoogleAuth<T extends AuthClient = JSONClient> {
 
     const targetScopes = this.getAnyScopes() ?? [];
 
-    const client = new Impersonated({
+    return new Impersonated({
       ...json,
-      delegates: json.delegates ?? [],
```

Comment:
Why was the delegates field deleted here?

Reply:
It's handled in the constructor itself:
- https://github.com/googleapis/google-auth-library-nodejs/blob/dc043cea4f87eeb642c165b9781586996f09854d/src/auth/impersonated.ts#L123-L127

We can de-dupe here